mw-webapp/src/hooks/useClickOutside.ts

```diff
@@ -0,0 +1,26 @@
+import React, {useEffect} from "react";
```

Comment:
Please explain why do we need this file

Reply:
We already have some components that need to close when clicked from outside (modal window, sidebar) and we don't use links as external interfaces. What do you think about using some headless components (from radix for example? This should help us reduce the maintenance of all this code and logic.